mint/run/core/aws-sdk-ruby/aws-stub-tests.rb

```diff
@@ -353,21 +405,21 @@ def presignedPostWrapper(bucket_name, file_name, expires_in_sec, max_byte_size,
   # To be addressed. S3 API 'get_bucket_policy' does not work!
   # def getBucketPolicy(bucket_name)
   #   # Returns bucket policy
-  #   return @@s3.bucket(bucket_name).get_bucket_policy
+  #   return s3.bucket(bucket_name).get_bucket_policy
```

Comment:
Any reason the `@@` was removed from `@@s3` here? :slightly_smiling_face:

Reply:
Code is removed.